services/super-app-cms/Makefile

```diff
@@ -0,0 +1,33 @@
+.DEFAULT_GOAL:=help
+
+# The default Strapi port. See comment on `dev` before running `make dev`.
+PORT?=1337
+
+# Explicitly label phonies because we have at least one non-phony (`build`).
+.PHONY: dev clean args help
+
+##@ Dev
+
+# NOTE: This affects where the app runs, but NOT where its own routes point.
+# That is configured in config/server.js, which gets baked in at build-time, not
+# run-time. So to properly run on PORT in development, you have to run `build`
+# before `dev`, e.g. `make PORT=4000 build && make PORT=4000 dev`.
+
+dev: ## Run Strapi with autoReload enabled
+	PORT=$(PORT) npm run develop
```

Comment:
would another `init` command that runs `npm ci` and copies `.env.example` to `.env` be helpful here?

Reply:
That's a good idea, I'll add that.